backend/.env

```
@@ -0,0 +1,4 @@
+DB_HOST=localhost
+DB_USER=postgres
+DB_PASSWORD=Spickles@004.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)